InnerEye/ML/configs/classification/CovidModel.py

```diff
@@ -234,24 +246,38 @@ def get_output_csv_path(mode: ModelExecutionMode) -> Path:
             p = get_best_epoch_results_path(mode=mode, model_proc=model_proc)
             return self.outputs_folder / p / MODEL_OUTPUT_CSV
 
+        def get_labels_and_predictions(df: pd.DataFrame) -> pd.DataFrame:
+            labels = []
+            predictions = []
+            for i, target in enumerate(self.target_names):
+                predictions.append(df[df[LoggingColumns.Hue.value] == target][LoggingColumns.ModelOutput.value].item())
+                labels.append(df[df[LoggingColumns.Hue.value] == target][LoggingColumns.Label.value])
```

Comment:
I'd define a variable `target_df = df[df[LoggingColumns.Hue.value] == target]` to make it a bit more readable.